Libraries/Azure.psm1

```diff
@@ -213,57 +205,179 @@ Function Validate-SubscriptionUsage($RGXMLData, $Location, $OverrideVMSize, $Sto
     }
 }
 
+Function Get-StorageAccountFromRegion($Region, $StorageAccount)
+{
+	#region Select Storage Account Type
+	$RegionName = $Region.Replace(" ","").Replace('"',"").ToLower()
+	$regionStorageMapping = [xml](Get-Content .\XML\RegionAndStorageAccounts.xml)
+	if ($StorageAccount) {
+		if ($StorageAccount -imatch "^ExistingStorage_Standard") {
+			$StorageAccountName = $regionStorageMapping.AllRegions.$RegionName.StandardStorage
+		}
+		elseif ($StorageAccount -imatch "^ExistingStorage_Premium")
+		{
+			$StorageAccountName = $regionStorageMapping.AllRegions.$RegionName.PremiumStorage
+		}
+	}
```

Comment:
} else {
how is this false block exercised? I don't see any logic reaching here.

Reply:
I don't see either. Just keep those previous code implementation, no harm, and more robustness. so be easy for that.